pyodbc.cpython-39-aarch64-linux-gnu.so,sha256=OBN5VynSjjJCPOvhio_EKkcTFR8hM3jwOIIBvrk9htY,989536
pyodbc.pyi,sha256=JWNpdgmrxhNioTjW6OoAEcXELcGZobcMYzKtYlcJRuY,32320
pyodbc-4.0.39.dist-info/WHEEL,sha256=WmgP0K1kdqJz8z1pv4uWmQ_zr3BEvqOuA4KvS4Wklqs,150
pyodbc-4.0.39.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.39.dist-info/RECORD,,
pyodbc-4.0.39.dist-info/METADATA,sha256=Cu1OPw2btz_Dj3huRmD0y9-0CLuocCtBc9rxYLBxYyc,3160
pyodbc-4.0.39.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
